lib/licensee/project_files/license_file.rb

```diff
@@ -20,6 +20,11 @@ def self.name_score(filename)
         return 0.5 if filename =~ /licen[sc]e/i
         0.0
       end
+
+      # case-insensitive block to determine if the given file is LICENSE.lesser
+      def self.lesser_gpl_score(filename)
+        filename.casecmp('license.lesser').zero? ? 1 : 0
```

Comment:
FWIW https://github.com/search?utf8=%E2%9C%93&q=filename%3Alicense.lesser finds 1519 results. 10x less than copying.lesser, but _maybe_ enough to handle. licence.lesser only 44.